improver/constants.py

```diff
@@ -83,3 +83,7 @@
 
 #: Dry Adiabatic Lapse Rate (DALR; K m-1)
 DALR = -0.0098
+
+#: Environmental Lapse Rate (ELR; K m-1)
+#: Also known as Standard Adiabatic Lapse Rate
```

Comment:
I've seen it referred to as standard adiabatic lapse rate in places e.g. [this paper](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/wrcr.20140) although to try to avoid any confusion, I've renamed this throughout to be "standard atmosphere lapse rate".

Reply:
Following a conversation, I've reverted this back to Environmental Lapse Rate for clarity.